composer.json

```diff
@@ -20,21 +20,17 @@
         "ext-json": "*",
         "barryvdh/laravel-debugbar": "^3.6",
         "barryvdh/laravel-ide-helper": "^2.10",
-        "codeception/codeception": "^4.1",
-        "codeception/module-asserts": "^2.0",
-        "codeception/module-laravel": "^2.1 | *@dev",
+        "codeception/codeception": "^4.1 | ^5.0.0-RC2",
+        "codeception/module-asserts": "^2.0 | ^3.0",
+        "codeception/module-laravel": "^2.1 | ^3.0 | *@dev",
         "facade/ignition": "^2.17",
         "fakerphp/faker": "^1.17",
-        "nunomaduro/collision": "^5.10",
-        "phpunit/phpunit": "^9.5"
+        "nunomaduro/collision": "^5.10"
     },
     "config": {
         "optimize-autoloader": true,
         "preferred-install": "dist",
-        "sort-packages": true,
-        "platform": {
-            "php": "7.4.0"
-        }
```

Comment:
this code forces composer to install versions of the dependencies compatible with this version of PHP, which is the minimum required by the module. I will revert this change.